docs/developer-guide/syntax-reference.rst

```diff
@@ -218,6 +218,10 @@ The WITH clause supports the following properties:
 | VALUE_FORMAT (required) | Specifies the serialization format of the message value in the topic. Supported formats:   |
 |                         | ``JSON``, ``DELIMITED`` (comma-separated value), and ``AVRO``.                             |
 +-------------------------+--------------------------------------------------------------------------------------------+
+| DELIMITER_FORMAT        | Used when VALUE_FORMAT='DELIMITED'. Supported formats:   DEFAULT, EXCEL, INFORMIX_UNLOAD,  |
```

Comment:
Remove extra space: "Supported formats:  DEFAULT..."

Reply:
fixed in ef32d83e..e0c07d2a